### PR TITLE
guake: add page

### PR DESCRIPTION
**Pull request by danielmichaels**
guake: add page

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/guake.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# guake
+
+> Drop-down terminal for GNOME.
```
Comment (pxgamer) [inline pages/linux/guake.md]:
```suggestion
> A drop-down terminal for GNOME.
```

File: pages/linux/guake.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Toggle Guake Visiblity:
+
+`F12`
+
+- Toggle Fullscreen:
+
+`F11`
+
+- Open New Tab:
```
Comment (pxgamer) [inline pages/linux/guake.md]:
```suggestion
- Open a new tab:
```

File: pages/linux/guake.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`Ctrl+Shift+T`
+
+- Close Terminal:
+
+`Super+X`
+
+- Goto Previous Tab:
+
+`Ctrl+Page Up`
```
Comment (mebeim) [inline pages/linux/guake.md]:
```suggestion
`Ctrl+PageUp`
```

File: pages/linux/guake.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# guake
+
+> Drop-down terminal for GNOME.
+
+- Toggle Guake Visiblity:
```
Comment (mebeim) [inline pages/linux/guake.md]:
```suggestion
- Toggle Guake visiblity:
```

File: pages/linux/guake.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+`F12`
+
+- Toggle Fullscreen:
```
Comment (mebeim) [inline pages/linux/guake.md]:
```suggestion
- Toggle fullscreen mode:
```

File: pages/linux/guake.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+`F11`
+
+- Open New Tab:
+
+`Ctrl+Shift+T`
+
+- Close Terminal:
```
Comment (mebeim) [inline pages/linux/guake.md]:
```suggestion
- Close the terminal:
```

File: pages/linux/guake.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+`Super+X`
+
+- Goto Previous Tab:
```
Comment (mebeim) [inline pages/linux/guake.md]:
```suggestion
- Go to the previous tab:
```

File: pages/linux/guake.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+`Ctrl+Page Up`
+
+- Search Selected Text in Broswer:
```
Comment (mebeim) [inline pages/linux/guake.md]:
```suggestion
- Search the selected text in the browser:
```

File: pages/linux/guake.md (hunk 2 above)
Comment (danielmichaels) [reply pages/linux/guake.md]: Thanks for the advice, it does read much better. :thumbsup:

**Comment by pxgamer** [comment]
Thanks @danielmichaels, LGTM!

**Pull request merged**